File: Windows/Java/static关键字/src/com/str/sta/Student.java
package com.str.sta;
public class Student{
    String name;
    static int num=0;
    public Student(String name){
        this.name = name;
        this.num++;
    }
    void Str(){
        System.out.println(this.name+"是第"+this.num+"个学生");
    }
}

File: Code/C#/第十天_/08继承/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _08继承
{
    public class Student : Person
    {
        public Student(string name,int age,char gender,string zhiwu) : base(name,age,gender)
        {
            this.ZhiWu = zhiwu;
        }
        private string _zhiWu;
        public string ZhiWu
        {
            get { return _zhiWu; }
            set { _zhiWu = value; }
        }
    }
}
